### PR TITLE
Update Helm to 2.16.9

### PR DESCRIPTION
**Pull request by cpanato**
**What this PR does / why we need it**:
- update HELM_VERSION to v2.16.9
- alpine to 3.12
- kubectl to 1.18.6
- YAMALE to3.0.2
- YAML_LINT to 1.24.2

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,31 +1,31 @@
-FROM alpine:3.11
+FROM alpine:3.12
 
 RUN apk --no-cache add \
     curl \
     git \
     libc6-compat \
     openssh-client \
-    python \
+    python3 \
```
Comment (unguiculus) [inline Dockerfile:8]: Just to make sure. Did you test this? Are the tools compatible? How about the other Python dependencies: Are those still fine?

**Comment by smlx** [comment]
Hi, just wanted to check on the status of this PR.

I'm hitting problems with the old version of helm in the chart-testing image that the chart-testing action uses by default. Specifically, the `lookup` function is failing validation (https://github.com/helm/helm/issues/7955). This is fixed in helm 3.2.x.

File: Dockerfile (hunk 1 above)
Comment (cpanato) [reply Dockerfile]: sorry for the delay, I've run the commands and after my latest change seems to work

**Comment by cpanato** [comment]
@unguiculus @davidkarlsen followed the approach that we have for master branch

**Pull request merged**